.github/workflows/bench.yml

```diff
@@ -0,0 +1,49 @@
+on: [ push, pull_request ]
+
+name: Benchmark
+
+jobs:
+  benchmark:
+    name: Run benchmarks
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest]
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
```

Comment:
Looks like there's a `v3` of `actions/checkout`

Reply:
we should upgrade across the board, but it seems most repos in our org use `v2`, at least that I've seen. will leave as-is, and take a broader TODO for updates throughout other repos too.